Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml

```
@@ -812,6 +813,10 @@
         - state: mag-twomode2-4
           map: ["enum.GunVisualLayers.TwoModeSecond"]
           shader: unshaded
+        - state: aim_module
+          visible: false
+          sprite: White/Objects/Weapons/modulesOnEGUN.rsi
+          map: [ "enum.ModuleVisualState.AimModule" ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)